client/src/components/TaskDetails.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+
+// Change this to modify the default placeholder
+const default_emptyContentPlaceholder: string = "N/A";
+
+// Mapping between database column names and display subheading titles to reduce further data processesing
+const sectionNamesMap: { [key: string]: string } = {
+  category: "Task Category",
+  title: "Task Title",
+  created_at: "Date",
+  suburb: "Suburb",
+  state: "State",
+  estimated_time: "Estimated Time", // Missing ERD column name (fix later when db added)
+  budget: "Estimated Price",
+  description: "Task Details",
+};
+
+// Type of the section data to be passed
+type SectionData = {
+  [key: string]: string | null; // Allow database column to have null for item (prints 'N/A')
+};
+
+// Props for TaskDetails (and the sections)
+type TaskDetailsProps = {
+  data: SectionData;
+  emptyContentPlaceholder?: string; // OPTIONAL: What to show if missing content for a subheading.
+};
+
+const TaskDetailsSection = ({
+  data,
+  emptyContentPlaceholder = default_emptyContentPlaceholder,
+}: TaskDetailsProps) => {
+  return (
+    <>
+      {Object.keys(sectionNamesMap).map((dbName, index) => (
+        <div
+          key={index}
+          className="py-[0.65rem] text-[1rem] font-[350] leading-[1.125rem] text-[#485358]"
```

Comment:
Don't use `[...]`, try sticking to the CSS font classes in `styles.css` (it was pushed less than an hour ago) and the preset tailwind classes like `py-3`. Having too many classes with custom sizes can get messy really quickly